lib/utils/build-ui-table.js

```diff
@@ -0,0 +1,161 @@
+const path = require('path');
+
+/**
+ *
+ * @typedef {String} ThumbnailPath  - path to a thumbnail
```

Comment:
I did not review this, i suggest you rework this considering my remarks on other jsdocs